### PR TITLE
Add iperf command

### PR DESCRIPTION
**Pull request by AdriannaPienkowska**

### CODE REVIEW

**Comment by Ernold11** [comment]
Check List
----------
* [ ]  CI is passed
* [ ]  master is merged into current branch
* [ ]  public methods should be at the top of class, protected at bottom  
       example: `on_new_line()` is public, `_parse()` is protected
* [ ]  Name of methods should tell what its doing, not how are doing  
       example: `_parse_re_via_dev_proto_metric()` -> `_parse_via_dev_proto_metric()`
* [ ]  All parsing lines (on_new_line) should be covered by unit tests
* [ ]  Be sure that  COMMAND_OUTPUT_* / COMMAND_KWARGS_* / COMMAND_RESULT_* cover all parsing lines
* [ ]  Parsed line example, parsing regexp, parsing method should be next to each other
* [ ]  Using named groups inside regexp (like `r"^\s*(?P<ADDRESS>\S+)\s+via\s+(?P<VIA>\S+)”`) is preferred
* [ ]  Longer regexp are before shorter regexp
* [ ]  Resources for pytest should be declared as `@pytest.fixture` or `@pytest.yield_fixture`
* [ ]  There are no sensitive data (company specific) like: IP addr, passwords, commands, tools etc

File: moler/cmd/unix/iperf.py [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+
+from moler.cmd.unix.genericunix import GenericUnixCommand
+from moler.exceptions import CommandFailure
+from moler.exceptions import ParsingDone
+import re
+
+
+class Iperf(GenericUnixCommand):
+    def __init__(self, connection, options):
```
Comment (Ernold11) [inline moler/cmd/unix/iperf.py]: Add prompt, newline

File: moler/cmd/unix/iperf.py [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+[  3]  9.0-10.0 sec  26.4 MBytes   221 Mbits/sec
+[  3]  0.0-10.0 sec   265 MBytes   222 Mbits/sec
+xyz@debian:~$"""
+
+COMMAND_KWARGS_basic_client = {
+    'options': '-c 10.1.1.1'
+}
+
+COMMAND_RESULT_basic_client = {
+    'RESULT': ['Client connecting to 10.1.1.1, TCP port 5001', 'TCP window size: 16384 Byte (default)',
```
Comment (Ernold11) [inline moler/cmd/unix/iperf.py]: Please parse output line, split to dict with Interval, Transfer, Bandwidth keys(normalize units - i.e. Mbits/sec, Kbits/sec, bits/sec)

**Comment by Ernold11** [comment]
- Still no normalized units (please keep both),
- When you finish make necessary changes please write comment into pull request (ready to review)

File: moler/cmd/unix/iperf.py [hunk 3]
```diff
@@ -0,0 +1,351 @@
...
+[904]   0.0-10.0 sec   11.8 MBytes   9.86 Mbits/sec   2.618 ms   9/ 8409  (0.11%)
+xyz@debian:~$"""
+
+COMMAND_KWARGS_basic_server = {
+    'options': '-u'
+}
+
+COMMAND_RESULT_basic_server = {
+    'CONNECTIONS': {
+        'local 10.1.1.1 port 5001 connected with 10.6.2.5 port 32781': [{'Bandwidth': '9.84 Mbits/sec',
```
Comment (Ernold11) [inline moler/cmd/unix/iperf.py]: Still not unit normalized - see at Unix.ls command

**Comment by AdriannaPienkowska** [comment]
I have finished making changes.

File: moler/cmd/unix/iperf.py [hunk 4]
```diff
@@ -0,0 +1,436 @@
...
```
Comment (Ernold11) [inline moler/cmd/unix/iperf.py]: Almost done. Bandwith raw should be output from command(9.84 Mbits/sec), bandwith should be in basic units(10317987)

Comment (AdriannaPienkowska) [reply moler/cmd/unix/iperf.py]: Why? It would be inconsistent with how for example 'Transfer' is parsed.

Comment (Ernold11) [reply moler/cmd/unix/iperf.py]: Transfer of course should be parsed like Bandwidth.

**Pull request merged**